lib/functionTemplates/template.js

```diff
@@ -0,0 +1,28 @@
+const format = require('string-format');
```

Comment:
This requires doubling up braces, which are used in both the Python and the JavaScript templates!